src/vs/workbench/contrib/chat/browser/chatContentParts/chatProgressContentPart.ts

```diff
@@ -80,35 +78,6 @@ function shouldShowSpinner(followingContent: IChatRendererContent[], element: Ch
 	return isResponseVM(element) && !element.isComplete && followingContent.length === 0;
 }
 
-export class ChatWorkingProgressContentPart extends ChatProgressContentPart implements IChatContentPart {
```

Comment:
I think we still need this, we can just remove the `isPaused` check

Reply:
Thank you, makes sense. I brought it back